tests/blockchain/eth/entities/agents/test_staking_escrow_agent.py

```diff
@@ -297,27 +314,27 @@ def test_winding_down(agency, testerchain, test_registry, token_economics):
 
     assert not staking_agent.is_winding_down(staker_account)
     assert staking_agent.get_locked_tokens(staker_account, token_economics.minimum_locked_periods) != 0
```

Comment:
This check also should have `+1`, the idea is to see consecutive periods to prove last period of sub-stake
And smae for other checks in this tests `staking_agent.get_locked_tokens`

Reply:
Ah - Yeah missed those.

Reply:
:heavy_check_mark: